.env.sample

```diff
@@ -1,12 +1,12 @@
 # do not commit sensitive info in the repo
 
-REACT_APP_ENDPOINT=http://localhost:3000
-REACT_APP_API_URI=http://localhost:8000
-REACT_APP_DS_API=https://a-ds.storysquad.dev/	
-REACT_APP_DS_TOKEN=<ds api key>
-REACT_APP_S3_KEY=<aws s3 key>
-REACT_APP_S3_SECRET_KEY=<aws secret key>
-
-REACT_APP_AUTH0_DOMAIN=<domain url>
-REACT_APP_AUTH0_CLIENT_ID=<client id>
-REACT_APP_AUTH0_CALLBACK_URL= http://localhost:3000
\ No newline at end of file
+REACT_APP_API_URI	= http://localhost:3000
```

Comment:
Let's revert this file back to the way it is on main. We don't want spaces around the "=" character because it needs to be readable by the shell (bash, sh, zsh, etc)